components/create_comment/index.ts

```diff
@@ -80,7 +80,7 @@ function makeMapStateToProps() {
             codeBlockOnCtrlEnter: getBool(state, Preferences.CATEGORY_ADVANCED_SETTINGS, 'code_block_ctrl_enter', true),
             ctrlSend: getBool(state, Preferences.CATEGORY_ADVANCED_SETTINGS, 'send_on_ctrl_enter'),
             createPostErrorId: err.server_error_id,
-            readOnlyChannel: !isCurrentUserSystemAdmin(state) && channel.name === Constants.DEFAULT_CHANNEL,
+            readOnlyChannel: false,
```

Comment:
Not sure if this is correct, or if there are more changes to be made here.

Reply:
I wonder if we can just remove the readOnlyChannel prop from this component (`CreateComment`) and wherever it was used we default to the `false` value of the logic?

Reply:
Yeah, that's the weird thing: there appears to be no `readOnlyChannel` prop for this component, just `CreatePost`. I wonder if the functionality was just broken to begin with?

Reply:
I'm not sure if I understand because I can see the `readOnlyChannel` prop being used in the `CreateComment` component?

Reply:
Ah, I misunderstood your comment. You're right that we can just remove it: but I'm worried about the divergent behaviour between `CreatePost` and `CreateComment` here, given that the former actually had a `readOnlyChannel` prop that clients of `CreatePost` would sometimes set. Are we 100% certain that `CreateComment` should never provide a read-only experience?

Reply:
That's a good point! I'm going to read through the components a bit to understand the `readOnlyChannel` prop a bit better. I'll let you know what I think

Reply:
@lieut-data I'm under the impression we can remove that prop from both components. The CreatePost component is only used in one place and that prop is unused. The prop was introduced a few years ago (https://github.com/mattermost/mattermost-webapp/pull/1415) but wasn't even in use back then